app/controllers/tours_controller.rb

```diff
@@ -2,6 +2,16 @@ class ToursController < ApplicationController
   before_action :load_tour_details, :current_user, only: :show
 
   def show
+    @quantity = Booking.new
+
+    @reviews = @tour_details.reviews.to_a
+
+    @avg_rating = if @reviews.blank?
```

Comment:
dùng toán tử 3 ngôi nhé
@reviews.blank? ? 0 : @tour_details.reviews.average(:rating)